### DISCUSSION TITLE
question: milestone release with Slick 3.5.1

### DISCUSSION THREAD

**Discussion by andyczerwonka** [apache/pekko-connectors#660] [General]
I noticed that the main branch [already contains the 3.5.1 dependency](https://github.com/apache/pekko-connectors/blob/main/project/Dependencies.scala#L445), which is something we need before we start our migration.

![image](https://github.com/apache/pekko-connectors/assets/138898/ee57d99c-e405-47ff-9429-2684c90751c2)

It looks like the current 1.1.0-M0 release [still contains the 3.3.3 dependency](https://github.com/apache/pekko-connectors/blob/v1.1.0-M0/project/Dependencies.scala#L404). Is there a build we can use to start our migration and do you expect another milestone release with 3.5.1 in a near future?

**Comment by pjfanning** [comment]
* 1.1.0-M0 is just a git tag used to mark the start of the 1.1.0 dev (there is no release for this tag)
* 1.1.0-M1 will be the first actual release with changes
* this release should happen within the next couple of months
* there are snapshots published already
* you could copy the pekko-connectors-slick code and build it yourself if you don't want to use a snapshot

**Reply by andyczerwonka** [reply]
Perfect, thank you. We can start our migration against a snapshot.

**Comment by pjfanning** [comment]
v1.1.0-M1 has been released with slick 3.5.1 support - including a Scala 3 compatible release for slick
